### ISSUE TITLE
Recognize songs in other apps through a notification service

### ISSUE THREAD

**Issue by Rafee-M** [alexmercerind/audire#24]
Using the application from the notification drawer and recognizing songs in other apps would be really useful.

Screenshot from [Audile](https://github.com/aleksey-saenko/MusicRecognizer):
![Screenshot_2024-01-17-13-41-14-41_3822ac0e29cfec8970059bb21d01f05e](https://github.com/alexmercerind/audire/assets/69535896/2e70cd96-a7d5-4212-b526-a3cae9400883)

**Comment by SimoLRepo** [comment]
> Using the application from the notification drawer and recognizing songs in other apps would be really useful.

@Rafee-M I noticed that on Android this functionality exists but it simply interrupts the open app to open Audire, which shouldn't happen. It should (via notification) listen to audio from other apps and then show a recognition notification.

**Comment by Rafee-M** [comment]
> It should (via notification) listen to audio from other apps and then show a recognition notification.

Yes and [Audile](https://github.com/aleksey-saenko/MusicRecognizer) seems to behave that way as well. That's why I was thinking maybe it could be implemented. iirc Shazam can do the same thing as well

**Comment by alexmercerind** [comment]
Hello, this is on my track. This depends upon #13.

**Comment by roke-julian-lockhart** [comment]
Note that, like [`metabrainz/listenbrainz-android/issues/590#issuecomment-3409389154`](https://github.com/metabrainz/listenbrainz-android/issues/590#issuecomment-3409389154), the application shall need to cope with the notification being dismissed, because I don't believe that a notification can be undismissable anymore.

**Comment by alexmercerind** [comment]
Yeah... Android 14 made that change & many others that kinda make foreground notifications "less capable".
